jenkins-scripts/docker/lib/_generic_linux_compilation_build.sh.bash

```diff
@@ -55,7 +55,7 @@ cat >> build.sh << DELIM_BUILD_DEPS
       dependency_repo="osrf/${dep}"
     else
       # need to replace _ by -
-      dependency_repo="ignitionrobotics/${dep//_/-}"
+      dependency_repo="gazebosim/${dep//_/-}"
```

Comment:
Did you track down how we end up here? It's possible that there's `ignitionrobotics` hardcoded somewhere else still

Reply:
![image](https://user-images.githubusercontent.com/20265670/175456032-1a378a45-665c-40d4-9175-5dfaf453d92d.png)

It's just used for github cloning! Should be safe